torch/ao/quantization/observer.py

```diff
@@ -98,10 +98,10 @@ class ObserverBase(ABC, nn.Module):
         dtype: Quantized data type
     """
 
-    def __init__(self, dtype, is_dynamic=None):
+    def __init__(self, dtype, replacement_quant_is_dynamic=None):
```

Comment:
should we just set default to True, just to reduce the possible values of `replacement_quant_is_dynamic`